templates/industrial/index.html

```diff
@@ -0,0 +1,695 @@
+        <li>Leverage cutting-edge accelerated computer vision libraries</li>
+        <li>Run deep learning inference engines, natively or in containers</li>
+      </ul>
+      <ul class="p-inline-images u-no-margin--bottom u-align--left">
+      
+        <li class="p-inline-images__item u-no-margin--left">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/c8749268-logo-ros.svg",
+              alt="ROS",
+              height="88",
+              width="333",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-inline-images__logo"},
+            ) | safe
+          }}
+        </li>
+        <li class="p-inline-images__item u-no-margin--left">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/8da00274-open-robotics.png",
+              alt="Open Robotics",
+              height="88",
+              width="308",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-inline-images__logo"},
+            ) | safe
+          }}
+        </li>
+        <li class="p-inline-images__item u-no-margin">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/1ac0e434-Nvidia_Logo_Horizontal.svg",
+              alt="Nvidia",
+              height="88",
+              width="461",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-inline-images__logo"},
+            ) | safe
+          }}
+        </li>
+      </ul>          
+      <a href="/robotics">
+        Learn more&nbsp;&rsaquo;
+      </a>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/3d44acdd-Robotics_icon.svg",
+          alt="",
+          height="250",
+          width="248",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+
+  <!-- Exciting HMIs and POS -->
+  <div class="row u-equal-height">
+    <div class="col-6 u-vertically-center u-align--center u-hide--small">
+      {{
+        image(
+          url="https://assets.ubuntu.com/v1/32e0b197-digital-kiosk.svg",
+          alt="",
+          height="250",
+          width="178",
+          hi_def=True,
+        ) | safe
+      }}
+    </div>
+    <div class="col-6">
+      <h2>Exciting HMIs and POS</h2>
+      <p class="p-heading--four">Fast, fluid and designer-friendly</p>
+      <ul>
+        <li>Implement robust digital signage and kiosk solutions</li>
+        <li>Upgrade your terminals securely, with reliable OTA updates</li>
+        <li>Benefit from a rich hardware and software ecosystem</li>
+        <li>Millions of smart displays already deployed on Ubuntu</li>
+        <li>Create rich graphical interfaces with Qt and Flutter</li>
+      </ul>
+      <ul class="p-inline-images u-no-margin--bottom u-align--left">
```

Comment:
I think `p-inline-images--small` matches the design slightly closer and looks better.